README.md

```diff
@@ -34,6 +34,7 @@ source ~/.bashrc
 | facerec disable | Temporarily disable facerec |
 | facerec remove | Completely remove the facerec  |
 | facerec --help | Get the CLI info |
+| facerec version | Display installed version of facerec |
```

Comment:
Keep `facerec version` before `facerec --help`. And change the command to `facerec --version`.